tests/ui/format_args.rs

```diff
@@ -114,6 +115,16 @@ fn main() {
     println!("error: something failed at {}", my_other_macro!());
     // https://github.com/rust-lang/rust-clippy/issues/7903
     println!("{foo}{foo:?}", foo = "foo".to_string());
+    print!("{}", (Location::caller().to_string()));
+    print!("{}", ((Location::caller()).to_string()));
+    #[rustfmt::skip]
+    print!("{}", (Location::caller()
+        .
+        to_string()));
+    #[rustfmt::skip]
+    print!("{}", ((Location::caller())
+        .
+        to_string()));
```

Comment:
Can remove these since they're the same as the above two aside from formatting

```suggestion
```

Reply:
@Alexendoo I made it this way on purpose because I wanted to ensure that the surround spacing is also captured.  I didn't want to just grab the dot, i wanted to grab everything after the ending of the prior expression.

Reply:
Sorry for the delay! 😅 

Generally we don't need to worry about whitespace to that degree as rustfmt will take care of it